core/domain/suggestion_registry_test.py

```diff
@@ -27,15 +29,60 @@
 (suggestion_models,) = models.Registry.import_models([models.NAMES.suggestion])
 
 
+class MockBaseSuggestion(suggestion_registry.BaseSuggestion):
```

Comment:
Not sure if this is the best idea to test this situation. I can't think of anything else. Have you used this pattern elsewhere? (To test other base classes maybe).

Reply:
Hmm I cannot think of any other way to test this. I don't think I used exactly this type of pattern anywhere but the base classes are tested in this manner:
https://github.com/oppia/oppia/blob/9b5d43d18d8644a9f652788d0b4e07f8d3287fe5/core/domain/value_generators_domain_test.py#L41

Reply:
But in those cases there is an implemented init for the base right? I am ok with how you have done it then.

Reply:
Also, @seanlip when you get a chance, could you advice on this pattern?